.ci/minikube_integration.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+#
+# Copyright Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: This script assumes that minikube is installed and running, and using the docker driver on Linux
+# Due to networking issues with the docker driver and ingress on macOS/Windows, this script must be run on Linux
+
+# Share docker env with Minikube
+eval $(minikube docker-env)
+
+# error on unset variables
+set -u
+# print each command before executing it
+set -x
+
+# Build the registry operator image
+export IMG=${REGISTRY_OPERATOR}
+make docker-build
+if [ $? -ne 0 ]; then
+    echo "Error building registry operator image"
+    exit 1;
+fi
+
+# Install cert-manager
+make install-cert
```

Comment:
Just to note, we should probably update the cert-manager version in the Makefile.  It's been a while since we updated, so it would be good to track this in a separate issue.

Reply:
Will be addressed in https://github.com/devfile/api/issues/1320.